.github/workflows/release-clarity-vscode.yaml

```diff
@@ -0,0 +1,108 @@
+name: Build Release for Clarity VSCode
+on:
+  pull_request:
+  push:
+    tags:
+     - 'v*'
+    paths-ignore:
+     - '**/CHANGELOG.md'
+     - '**/package.json'
+     - '**/package-lock.json'
+    branches:
+     - main
+     - develop
+    paths:
+     - components/clarity-vscode
+     - components/clarinet-cli
+     - components/chainhook-event-observer
```

Comment:
Paths are incorrect

Reply:
Here's the complete list:
```
components/clarity-lsp
components/chainhook-types
components/clarinet-deployments
components/clarinet-files
components/clarinet-utils
components/clarity-repl
components/hiro-system-kit
components/stacks-rpc-client
```

Reply:
And `components/clarity-vscode` (which is the component we want to build)